scripts/zones/Mhaura/Zone.lua

```diff
@@ -14,6 +14,24 @@ require("scripts/globals/settings");
 require("scripts/globals/zone");
 -----------------------------------
 
+function onGameHour(zone)
+    -- Script for Laughing Bison sign flip animations
+    local timer = 1152 - ((os.time() - 1009810802)%1152);
+    local destination = 0; -- Selbina, set to 1 for Al Zhabi
+    local direction = 0; -- Arrive, 1 for depart
+    local waiting = 216; -- Offset for Selbina
+   
+    -- Next ferry is Al Zhabi for higher values.
+    if (timer >= 576) then
+        destination = 1;
+        GetNPCByID(17797183):AnimationSub(1)
```

Comment:
We're trying to avoid having entity ID's all over the place, you should add Laughing Bison to `MobIDs.lua`, make sure it's required in the file, then reference the variable name you gave him. 

EG;
MobIDs.lua
```lua
LAUGHING_BISON = 17797183
```

Then for this line, do

```lua
GetNPCByID(LAUGHING_BISON):AnimationSub(1)
```